openrouteservice/src/main/java/heigit/ors/routing/graphhopper/extensions/flagencoders/ORSAbstractFlagEncoder.java

```diff
@@ -38,9 +41,12 @@ protected ORSAbstractFlagEncoder(int speedBits, double speedFactor, int maxTurnC
 			if(interimNodes > 0) {
 				interimDistance = estDist/(interimNodes+1);
 			}
-			if(interimDistance < 100) {
+			if(interimDistance < 100 && speed > 10) {
```

Comment:
do we need to have this check? I ma just thinking that if we have a speed of say 7km/h, this will not get halved and so would stay at 7. But then a speed of 12km/h would be halved, meaning that it would then be seen as havng a slower speed than the 7km/h way. 

if we don't put this check in, then we would end up with 7/2 = 3.5km/h, but then in the nxt check (speed <5) it would be taken back up to 5 and so avoid the problems

Reply:
Done